storage/src/main/java/com/tencent/rss/storage/handler/impl/LocalFileWriteHandler.java

```diff
@@ -92,6 +92,14 @@ public synchronized void write(
     String dataFileName = ShuffleStorageUtils.generateDataFileName(fileNamePrefix);
     String indexFileName = ShuffleStorageUtils.generateIndexFileName(fileNamePrefix);
 
+    // if shuffle has been uploaded totally, the directory of shuffle will be deleted
+    // the event should be dropped.
```

Comment:
// if shuffle has been uploaded totally... -> // Ignore this write, if the shuffle directory is deleted after being uploaded in multi mode or after its app heartbeat times out.

And it may be better to put this check before the `System.currentTimeMillis()`

Reply:
OK